MATLAB/Utilities/VarianCBCT/ReadXim.m

```diff
@@ -0,0 +1,117 @@
+function img=ReadXim(filename, varargin)
+%   READXIM(FILENAME, {READ_PIXEL_DATA})
+%       reads XIM image files from Varian TrueBeam accelerators 
+%       and returns a struct containing the XIM image information
+%       and pixel data.
+%   
+%   FILENAME is the full path to the XIM-image file
+%   READ_PIXEL_DATA determines whether pixel data should be decoded
+%       0: skip
+%       1: decode  (default)
+%
+%   Developed by Fredrik Nordstr�m 2015
```

Comment:
I downloaded this .m file on Matlab File Exchange.
https://www.mathworks.com/matlabcentral/fileexchange/53367-readxim

License info:

> Copyright (c) 2015, Fredrik
All rights reserved.

> Redistribution and use in source and binary forms, with or without

> modification, are permitted provided that the following conditions are met:

> * Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.

> * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution

> * Neither the name of Lund University nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.

> THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

Reply:
Fantastic. This is a BSD 3 License, the same as TIGRE, and it means we can use it as long as we keep the license file and credit the original authors. Thanks for confirming.